### DISCUSSION TITLE
How to let Avalonia Desktop on Windows using System Font ?

### DISCUSSION THREAD

**Discussion by ismlsmile** [AvaloniaUI/Avalonia#14704] [Q&A] [answered]
The default font on windows is not good looking, and I do not want to include other fonts in the project, just want to use system font: YaHei. Is it possible ?

**Comment by maxkatz6** [comment, marked as answer]
You can set FontFamily property on a Window, or globally with a style like this:
```
<Style Selector=":is(TopLevel)">
    <Setter Property="FontFamily" Value="Your Font Name" />
</Style>
```

You can specify multiple fonts separated by comma, so it will have a fallback.
